lib/display.ex

```diff
@@ -53,6 +53,11 @@ defmodule Display do
     format_error(error) |> IO.puts
   end
 
+  def congratulate do
+    format_green("\nYou have learned much. You must find your own path now.")
```

Comment:
I would either pull the `IO.puts` onto the same line or pipe everywhere in the file, just for consistency.